core/src/main/scala/pureconfig/ConfigCursor.scala

```diff
@@ -121,6 +175,7 @@ sealed trait ConfigCursor {
    * @return a `Right` with this cursor as a list or object cursor if the cast can be done, `Left` with a list of
    *         failures otherwise.
    */
+  @deprecated("Use `asListCursor` and/or `asCollectionCursor` instead", "0.10.0")
```

Comment:
You meant `asListCursor and/or asObjectCursor` here, right?

Reply:
Yes, of course. Fixed in 9f33a80.